test/testground/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/celestiaorg/celestia-app/test/testground
 go 1.21.1
 
 require (
-	github.com/celestiaorg/celestia-app v1.0.0-rc0.0.20231120210508-c4a02614e8bc
+	github.com/celestiaorg/celestia-app v1.0.0-rc0.0.20231219172024-f57f5c104ea0
```

Comment:
It seems like the fix was to upgrade to a more recent version of celestia-app. A.k.a this commit: https://github.com/celestiaorg/celestia-app/commit/f57f5c104ea03b644bb6e50e262f212036eba99e

Reply:
Yes, that is correct.